packages/common/src/sites/HubSites.ts

```diff
@@ -461,6 +461,19 @@ export async function fetchSite(
   // get the model
   const model = await fetchSiteModel(identifier, requestOptions);
 
+  return convertModelToSite(model, requestOptions);
```

Comment:
Simply refactored this into a separate fn, callable when the app already has an IModel